src/main/kotlin/org/jlleitschuh/gradle/ktlint/Reporter.kt

```diff
@@ -37,8 +37,8 @@ fun JavaExec.applyReporter(target: Project, extension: KtlintExtension) {
 }
 
 private fun isReportAvailable(version: String, availableSinceVersion: String): Boolean {
-    val versionsNumbers = version.split('.')
-    val reporterVersionNumbers = availableSinceVersion.split('.')
+    val versionsNumbers = version.split('.').map { it.toInt() }
+    val reporterVersionNumbers = availableSinceVersion.split('.').map { it.toInt() }
```

Comment:
@shyiko Just want to check with you, do you think you will ever have a release that has some sort of `RC1` specifier on the end?
If not this should be fine.

Reply:
@JLLeitschuh I may apply your proposal to use symver plugin in a separate PR

Reply:
@JLLeitschuh never (and you can count on it). ktlint adheres to strict semver without pre-release|s or build meta.